### ISSUE TITLE
editorial parenthesis within a text

### ISSUE THREAD

**Issue by gcelano** [PerseusDL/canonical-latinLit#320]
As far as I can see, there are cases where an editorial expunction in the main text is kept as it is in the print edition (e.g., [ vidit ]) and cases where the square brackets are translated into TEI markup (e.g., <del>vidit</del>). Should be try to uniformize to the latter?

**Comment by lcerrato** [comment]
@gcelano 
Yes, I would try to remove all similar apparatus markings. The inconsistencies are a big problem with the Latin where things were converted in batches and there was no comprehensive content review. As I review texts, I try to catch these globally, but I am mainly working in Greek at present.
